nodes/src/nodes/approval/approval_manager.py

```
@@ -0,0 +1,232 @@
+            reviewer: Identity of the reviewer.
+            comment: Optional reviewer comment.
+
+        Returns:
+            The updated approval request dict.
+
+        Raises:
+            KeyError: If *approval_id* is not found.
+            ValueError: If the request is not in 'pending' status.
+        """
+        if self._require_comment and not comment:
+            raise ValueError('A comment is required when approving (require_comment is enabled)')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)